src/Checkbox/sgds-checkbox.ts

```diff
@@ -0,0 +1,69 @@
+import { LitElement, html } from "lit";
+import { customElement, property, query } from "lit/decorators.js";
+import { live } from "lit/directives/live.js";
+import styles from "./sgds-checkbox.scss";
+import { ifDefined } from "lit/directives/if-defined.js";
+import SgdsElement from "../utils/sgds-element";
+import { watch } from "../utils/watch";
+import { classMap } from "lit/directives/class-map.js";
+
+@customElement("sgds-checkbox")
+export class SgdsCheckbox extends SgdsElement {
+  static styles = styles;
+
+  @query('input[type="checkbox"]') input: HTMLInputElement;
+
+  /** Name of the HTML form control. Submitted with the form as part of a name/value pair. */
+  @property() name: string;
+
+  /** Value of the HTML form control. Primarily used to differentiate a list of related checkboxes that have the same name. */
+  @property() value: string;
+
+  /** Makes the checkbox a required field. */
+  @property({ type: Boolean, reflect: true }) required = false;
+
+  /** Draws the checkbox in a checked state. */
+  @property({ type: Boolean, reflect: true }) checked = false;
+
+  /** Disables the checkbox (so the user can't check / uncheck it). */
+  @property({ type: Boolean, reflect: true }) disabled = false;
+
+  /** Simulates a click on the checkbox. */
+  click() {
+    this.input.click();
+  }
+
+  handleClick() {
+    this.checked = !this.checked;
```

Comment:
somehow trying to get the e.target.value using @click event will return two values (initial and latest). I think click event runs twice ( the native click event and the custom handleClick() ) 

Dispatch a change event (lets say called "sgds-change"),  when this.emit('sgds-change') event changes. Either below line 37 
```this.emit('sgds-change')```

or 

create a function that uses @watch to watch checked and runs the emit method 

```
 @watch('checked', { waitUntilFirstUpdate: true })
  emitOnChange() {
   this.emit('sgds-change')
   }
```

and document this event name on storybook argstable under Event category